HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -2253,6 +2253,21 @@ def self.apply_hvac(runner, hpxml_bldg, weather, convert_shared_systems, unit_nu
       end
     end
 
+    # Defrost Backup Heat
+    hpxml_bldg.heat_pumps.each do |heat_pump|
+      next unless heat_pump.backup_heating_active_during_defrost.nil?
+      next unless [HPXML::HVACTypeHeatPumpAirToAir,
+                   HPXML::HVACTypeHeatPumpMiniSplit,
+                   HPXML::HVACTypeHeatPumpRoom,
+                   HPXML::HVACTypeHeatPumpPTHP].include? heat_pump.heat_pump_type
+
+      # The input is only used when there's integrated backup heat
+      next unless heat_pump.backup_type == HPXML::HeatPumpBackupTypeIntegrated
+
+      heat_pump.backup_heating_active_during_defrost = !heat_pump.distribution_system.nil?
```

Comment:
If no backup heat or "separate" backup heat, `backup_heating_active_during_defrost` will be nil and backup heat will never be modeled during defrost.

If "integrated" backup heat, defaults to true for ducted and false for ductless systems, with the ability for the user to override it on a per-system basis.